mods/ts/sequences/bridges.yaml

```diff
@@ -0,0 +1,167 @@
+lobrdg01:
```

Comment:
That only works if the sprite image is identical. For example:

```yaml
tracks01:
	idle:
		UseTilesetExtension: true
		ZOffset: -1c0

tracks02:
	Inherits: tracks01
```

etc. won't get rendered.